_data/tools.yml

```diff
@@ -535,6 +535,14 @@
   v2: false
   v3: true
 
+- name: openapi3
+  category: parsers
+  github: https://github.com/Dorthu/openapi3
+  language: Python
+  description: "A Python OpenAPI 3 Specification client and validator for Python 3."
```

Comment:
could you specify which type of validator? We've coined "data validator" and "schema validator" depending on wether its validating data against the schema, or the schema itself.

Reply:
This is primarily a schema validator, but it does validate responses if used to interact with an API.

Reply:
I'll update the PR.